File: 프로그래머스/2/64065. 튜플/튜플.py
def solution(s):
    answer = []
    s = s[1:len(s)-2].replace("{", "").split("},")
    s = sorted(s, key = lambda x : len(x))
    
    for items in s:
        items = items.split(",")
        for item in items:
            if (int(item) not in answer):
                answer.append(int(item))

    return answer

File: 프로그래머스/2/64065. 튜플/튜플.js
function solution(s) {
    const answer = [];
    
    const formattedS = s.slice(2, s.length - 2).split("},{").map((i) => i.split(',').map((j) => Number(j)));
    const sortedS = formattedS.sort((a, b) => a.length - b.length)
    sortedS.forEach((row) => {
        row.forEach((item) => {
            if (!answer.includes(item)) answer.push(item)
        })
    })
    
    return answer;
}